doc/en/how-to/output.rst

```diff
@@ -15,6 +15,15 @@ Examples for modifying traceback printing:
     pytest --showlocals     # show local variables in tracebacks
     pytest -l               # show local variables (shortcut)
     pytest --no-showlocals  # hide local variables (if addopts enables them)
+    pytest --quiet          # quiet - less verbose - mode
```

Comment:
Those look nice, but I think they should appear in the `Verbosity` section instead: although they do affect tracebacks, seems like `Verbosity` is a better place to mention them.

Reply:
Will do. Would you like me to move all the lines that I have added to `Verbosity `?